target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq6000-gl-axt1800.dts

```diff
@@ -0,0 +1,169 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+/dts-v1/;
+
+#include "ipq6000-glinet.dtsi"
+
+/ {
+	model = "GL.iNet GL-AXT1800";
+	compatible = "glinet,gl-axt1800", "qcom,ipq6018";
+
+	aliases {
+		label-mac-device = &dp1;
+	};
+
+	vcc_sd: regulator-vcc-sd {
+		compatible = "regulator-fixed";
+		regulator-name = "vcc_sd";
+		regulator-min-microvolt = <1800000>;
+		regulator-max-microvolt = <1800000>;
+		gpios = <&tlmm 66 GPIO_ACTIVE_HIGH>;
+		enable-active-high;
+		regulator-always-on;
+	};
+
+	vcc_fan: regulator-vcc-fan {
+		compatible = "regulator-fixed";
+		regulator-name = "vcc_fan";
+		regulator-min-microvolt = <5000000>;
+		regulator-max-microvolt = <5000000>;
+		gpio = <&tlmm 29 GPIO_ACTIVE_HIGH>;
+		enable-active-high;
+		regulator-boot-on;
+	};
+
+	fan: pwm-fan {
+		compatible = "pwm-fan";
+		pwms = <&pwm 1 40000 0>;
+		fan-supply = <&vcc_fan>;
+		interrupt-parent = <&tlmm>;
+		interrupts = <31 IRQ_TYPE_EDGE_RISING>;
+		cooling-levels = <36 128 192 255>;
+		#cooling-cells = <2>;
+	};
+
+	thermal-zones {
+		cpu-thermal {
+			trips {
+				cpu_trip_high: active-high {
+					temperature = <100000>;
+					hysteresis = <2000>;
+					type = "active";
+				};
+
+				cpu_trip_med: active-med {
+					temperature = <75000>;
+					hysteresis = <2000>;
+					type = "active";
+				};
+
+				cpu_trip_low: active-low {
+					temperature = <50000>;
+					hysteresis = <2000>;
+					type = "active";
+				};
+			};
+
+			cooling-maps {
+				cpu-active-high {
+					cooling-device = <&fan 3 3>;
+					trip = <&cpu_trip_high>;
+				};
+
+				cpu-active-med {
+					cooling-device = <&fan 2 2>;
+					trip = <&cpu_trip_med>;
+				};
+
+				cpu-active-low {
+					cooling-device = <&fan 1 1>;
+					trip = <&cpu_trip_low>;
+				};
+			};
+		};
+	};
+};
+
+&tlmm {
+	pwm_pins: pwm-pins {
+		pwm {
+			pins = "gpio30";
+			function = "pwm13";
+			drive-strength = <8>;
+		};
+	};
+
+	sd_pins: sd-pins {
+		sd {
+			pins = "gpio62";
+			function = "sd_card";
+			bias-pull-up;
+		};
+		ldo {
+			pins = "gpio66";
+			function = "gpio";
+			bias-pull-up;
+		};
+	};
+};
+
+&partitions {
+	partition@a00000 {
+		label = "rootfs";
+		reg = <0x0a00000 0x7280000>;
+	};
```

Comment:
This should be
```
partition@a00000 {
		label = "ubi";
		reg = <0x0a00000 0x7280000>;
		compatible = "linux,ubi";
};
```
To get rid of these warning:
```
mtdsplit: no squashfs found in "rootfs"
```

```
mtdblock: MTD device 'rootfs' is NAND, please consider using UBI block devices instead.
```

And need to override bootarg from u-boot.